lib/vanagon/component/dsl.rb

```diff
@@ -217,7 +221,7 @@ def install_service(service_file, default_file = nil, service_name = @component.
         end
 
         # Register the service for use in packaging
-        @component.service = OpenStruct.new(:name => service_name, :service_file => target_service_file, :type => service_type)
+        @component.service << OpenStruct.new(:name => service_name, :service_file => target_service_file, :type => options[:service_type])
```

Comment:
this too

Reply:
Nope, that was a pre-existing option used for some packaging types.